specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-04-01-preview/workspaceRP.json

```diff
@@ -4190,6 +4980,133 @@
         "modelAsString": true
       }
     },
+    "RaiBlocklistConfig": {
+      "type": "object",
+      "properties": {
+        "blocking": {
+          "type": "boolean"
```

Comment:
consider using enum instead of boolean for all `boolean` type properties

Reply:
this has already been released and used by UX, will consider that in the future API design, but will keep it as boolean this time.